### PR TITLE
carthage 0.6.3

### PR DESCRIPTION
**Pull request by jspahrsummers**
https://github.com/Carthage/Carthage/releases/tag/0.6.3

### CODE REVIEW

File: Library/Formula/carthage.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Carthage < Formula
   homepage "https://github.com/Carthage/Carthage"
-  url "https://github.com/Carthage/Carthage.git", :tag => "0.6.2",
+  url "https://github.com/Carthage/Carthage.git", :tag => "0.6.3",
```
Comment (mikemcquaid) [inline Library/Formula/carthage.rb]: Can you add a `:revision => ` indicating the revision this tag points to? Thanks!

**Comment by jspahrsummers** [comment]
@mikemcquaid Done!

**Comment by tdsmith** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Comment by tdsmith** [comment]
Oops; closed in 031f53684.

**Pull request closed without merging**